File: program1_5.java
// Accept one number from user and print that times * on screen

import java.util.*;

class DisplayX
{
    public void Display(int iNo)
    {
        int iCnt=0;

        for(iCnt=1; iCnt<=iNo; iCnt++)
        {
            System.out.print(" * ");
        }
    }
}
class program1_5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter Number:");
        int iValue=sobj.nextInt();

        DisplayX dobj= new DisplayX();
        dobj.Display(iValue);
    }
}

File: Program1.5.c
///////////////////////////////////////////////////////////////////////////////////////////////
//  Problem statement:Accept one number from user and print that number of * on screen
//  Output:
//////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>

void Accept(int iNo)
{
    int iCnt=0;

    for(iCnt=1; iCnt<=5; iCnt++)
    {
        printf("*\t");
    }
}
int main()
{
    int iValue=5;

    Accept(iValue);

    return 0;
}